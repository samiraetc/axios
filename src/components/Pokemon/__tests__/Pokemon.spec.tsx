import { render } from '@testing-library/react';
import React from 'react';
import Pokemon from '../Pokemon';



describe('<Pokemon />', () => {
  it('should render', async () => {
  
    const { asFragment, getByText, queryByText } = render(
     <Pokemon />
    );
   
    expect(getByText('Pokemon')).toBeInTheDocument();
    expect(asFragment()).toMatchSnapshot();
  });
});
