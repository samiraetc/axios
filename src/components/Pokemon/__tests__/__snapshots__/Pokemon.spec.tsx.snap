// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Pokemon /> should render 1`] = `
<DocumentFragment>
  <div>
    Pokemon
  </div>
</DocumentFragment>
`;
